chart/values.yaml

```diff
@@ -14,9 +14,6 @@ certificates:
 image:
   pullPolicy: IfNotPresent
   pullSecrets: []
-  # community image
-  #repository: k8s.gcr.io/multitenancy/externalip-webhook
-  #tag: v1.0.0
   repository: rancher/externalip-webhook
   # Defaults to .Chart.appVersion
   # tag: ""
```

Comment:
This still needs an actual tag. The scripts which grabs images for images.txt needs it.

Reply:
Done